nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -216,20 +216,26 @@ in
       debug = mkEnableOption "gnome-session debug messages";
 
       flashback = {
-        enableMetacity = mkEnableOption "the standard GNOME Flashback session with Metacity";
+        enableMetacity = mkOption {
+          type = types.bool;
+          default = true;
+          example = "false";
+          description = "Whether to enable the standard Metacity GNOME flashback session.";
+        };
 
         customSessions = mkOption {
           type = types.listOf (types.submodule {
             options = {
+              # Deprecated, kept here just to show an error later.
               wmName = mkOption {
-                type = types.str;
-                description = "The filename-compatible name of the window manager to use.";
-                example = "xmonad";
+                type = types.unspecified;
+                visible = false;
+                default = null;
               };
 
               wmLabel = mkOption {
                 type = types.str;
-                description = "The pretty name of the window manager to use.";
+                description = "The display name of the window manager to show in the session.";
```

Comment:
What do you mean “in the session”? Will not that just be visible in the display manager?

Reply:
Done.